.github/workflows/linting-formatting.yaml

```diff
@@ -13,11 +13,19 @@ jobs:
     - name: Check out repo
       uses: actions/checkout@v2
 
-    - name: Install black
+    - name: Install black and tox
       run: |
         set -eux
         sudo apt-get install python3-setuptools
+        pip install tox
```

Comment:
Might be worth installing this via `apt` instead, since the version looks recent enough and we don't have to worry about `pip` vs `pip3`.